backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -234,6 +239,45 @@ def test_set_project_link(self):
         project.refresh_from_db()
         self.assertEqual(project.link_submission, link_2)
 
+    @mock.patch.object(RecruitProject, "setup_repository")
+    def test_setup_project_repo_call_from_api_action_option(self, setup_repository):
+
+        JAVASCRIPT = "js"
+        super_user = factories.UserFactory(is_superuser=True)
+
+        def get_repo_mock(
```

Comment:
Where is this used? Are you sure you need it?

Reply:
I used it before, but you're right, I don't need it anymore so I got rid of it.